### PR TITLE
LOG-435. Enable CI for openshift/elasticsearch-proxy

### PR DESCRIPTION
**Pull request by jcantrill**
For https://jira.coreos.com/browse/LOG-435

### CODE REVIEW

**Comment by jcantrill** [comment]
> oh. it's probably the Makefile change. do you really intend that?

reverted Makefile change

**Comment by bparees** [comment]
ah well i can't approve `cluster` dir changes either.  you'll need @stevekuznetsov or someone from his team.

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -1 +1,815 @@
+<<<<<<< Updated upstream
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: You have merge conflicts here ... don't change these files.

File: ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```
[the remark this answers is not in the document]
Comment (jcantrill) [reply ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml]: bumped to 1.12.. is that sufficient?

File: core-services/prow/02_config/_config.yaml [hunk 3]
```diff
@@ -170,6 +170,7 @@ tide:
     - openshift/dedicated-admin-operator
     - openshift/descheduler-operator
     - openshift/elasticsearch-operator
+    - openshift/elasticsearch-proxy
```
Comment (stevekuznetsov) [inline core-services/prow/02_config/_config.yaml]: This only adds merging for release branches -- also add to the other list so you get `master` as well

Comment (jcantrill) [reply core-services/prow/02_config/_config.yaml]: @stevekuznetsov did i update this file correctly?

**Comment by jcantrill** [comment]
@stevekuznetsov bump... is there someone else who can also do what is necessary?

**Pull request merged**